pkg/server/server.go

```diff
@@ -442,6 +452,13 @@ func setControlPlaneRoleLabel(ctx context.Context, nodes v1.NodeClient) error {
 		}
 		node.Labels[ControlPlaneRoleLabelKey] = "true"
 		node.Labels[MasterRoleLabelKey] = "true"
+
+		// remove etcd taint if exists
```

Comment:
I thought we weren't adding taints any more, should this remove the label instead?

Reply:
fixed